src/index.ts

```diff
@@ -0,0 +1,11 @@
+import * as _ from 'lodash';
```

Comment:
Why do you have 2 index files here?

Reply:
Sorry I forgot remove one of those files